packages/shared/src/lib/tracking.ts

```diff
@@ -32,9 +37,12 @@ export const addTrackingParams = (baseUrl: string, params: Tracking): string =>
         REFPVID: params.ophanPageId || 'not_found',
         INTCMP: params.campaignCode || '',
         acquisitionData: acquisitionData,
+        numArticles,
     };
 
-    const queryString = Object.entries(trackingLinkParams).map(([key, value]) => `${key}=${value}`);
+    const queryString = Object.entries(trackingLinkParams)
+        .filter(([, value]) => value !== undefined)
+        .map(([key, value]) => `${key}=${value}`);
```

Comment:
This logic is for filtering out article count from the url if the value is undefined